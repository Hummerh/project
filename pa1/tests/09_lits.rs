b'H' b'x' b'Y' b'%' b'$' b'#'
b"Hello world !?"
b"123_^_!@#$%^^&*()abc"


