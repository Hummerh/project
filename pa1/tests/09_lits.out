LITCHAR(H)
LITCHAR(x)
LITCHAR(Y)
LITCHAR(%)
LITCHAR($)
LITCHAR(#)
LITSTR(Hello world !?)
LITSTR(123_^_!@#$%^^&*()abc)
